src/App.css

```diff
@@ -50,6 +50,107 @@ ul {
 
 .navbar {
   position: absolute;
-  bottom: 10px;
+  bottom: 0px;
   width: 100%;
 }
+
+.navbar-items {
+  columns: 2;
+}
+
+.navbar-item-add {
+  background-color: rgb(206, 236, 129);
+  padding: 25px;
+  width: 100%;
+}
+
+.navbar-item-view {
+  background-color: rgb(245, 248, 131);
+  padding: 25px;
+  width: 100%;
+}
+
+.navbar-item-add .navbar-item-view {
+  position: relative;
+  display: inline-block;
+}
+
```

Comment:
Really nice to have: you have emotion in your files, so, for example, you could have the navbar stuff in navbar.js. You can use sass rules to make it nice. 
Consider using hex for color values. (this could be a personal preference of mine. I find they're faster to write and easier to understand).
Consider having css variables or a js variables file (if you go with emotion). Think about what you would change if you had a theme? (text-color, bgcolor, images)

Reply:
I can convert our palette to hex color values. 
I have never used CSS variables--have read about them and would be willing to experiment.
Also looking for feedback on our palette for an app like this (are colors too similar? too much green?)

Reply:
Here's some info on theming, css variables and emotion https://epicreact.dev/css-variables/